ENIGMAsystem/SHELL/Platforms/xlib/XLIBwindow.cpp

```diff
@@ -113,7 +119,15 @@ bool initGameWindow()
   if (winh > screen->height) winh = screen->height;
 
   // Make the window
+  #if !defined(GRAPHICS_NONE)
```

Comment:
These macros are going to cause issues on system changes you need to conditionally build them and stubs using the makefiles instead

Reply:
I've done what you said locally. I'll create a new pull request and close this one.